DOCSUP-9861: Date and time operator update (#25236)
ClickHouse/ClickHouse: docs/en/sql-reference/operators/index.md

└─────────────────────┴────────────────────────────────────────────────────────────┘
```

> You can set the period without using `INTERVAL` by multiplying seconds, minutes, and hours. For example, a period of one day can be set at `60*60*24`.

Reviewer: ```suggestion
You can set the date without using `INTERVAL`, just by adding or subtracting seconds, minutes, and hours. For example, an interval of one day can be set by adding `60*60*24`.
```

!!! note "Note"

>     Syntax `now() + 60*60*24` doesn't consider time settings. For example, daylight saving time. The `INTERVAL` syntax or `addDays`-function is always preferred.

Reviewer: ```suggestion
    Syntax `now() + 60*60*24` doesn't consider time settings. For example, daylight saving time. The `INTERVAL` syntax or `addDays` function are always preferred.
```

Examples:

``` sql

> SELECT now() AS current_date_time, current_date_time + 60*60*96;

Reviewer 2: What if we add better example with time shift?
```sql
:) SELECT toDateTime('2014-10-26 00:00:00', 'Europe/Moscow')  AS time, time + 60*60*24 as time_plus_24_hours, (time + interval 1 day) as time_plus_1_day

SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + ((60 * 60) * 24) AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day

Query id: aaf32215-aa05-4a37-a370-b85b73aaf386

┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```

(`2014-10-26 02:00:00` msk was time shift -1 hour)

Author: Great example, thanks.

> SELECT now() AS current_date_time, current_date_time + 60*60*96;

Reviewer 3: ```suggestion
SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + 60 * 60 * 24 AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day
```

```

``` text

> ┌───current_date_time─┬─plus(now(), multiply(multiply(60, 60), 96))─┐
> │ 2021-06-12 17:34:49 │                         2021-06-16 17:34:49 │
> └─────────────────────┴─────────────────────────────────────────────┘

Reviewer 3: ```suggestion
┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```
